### PR TITLE
Display event ID & make banner clickable

### LINKED ISSUE #427: Event Register - Add RFA ID from beacon job when it's created from beacon
make it the first column heading.

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
Display event ID in supertext.
Make job banner tap go to job system.
Fix callback not creating event identifier 
Closes #427

### CODE REVIEW

**Comment by OSPFNeighbour** [comment]
screenshot 2018-05-14 16 01 24

File: client/src/screens/events/Detail.js [hunk 1]
```diff
@@ -362,7 +362,7 @@ class Detail extends Component {
                 title={event.name}
                 subtitle={event.details}
                 icon="external-link"
-                onPress={() => undefined}
+                onPress={() => { Linking.openURL(event.permalink); }}
```
Comment (sdunster) [inline client/src/screens/events/Detail.js]: please move this to a component instance method so you can save a re-render every time it updates (which is at least every 5 seconds)
```
onPress={this.openExternal}
```

**Comment by OSPFNeighbour** [comment]
#427

**Comment by sdunster** [comment]
here I'll update the description for you

**Pull request merged**